### PR TITLE
Add metrics explore panel resize functionality

### PR DESCRIPTION
**Pull request by arsengit**
- Enabled resize handler on `Box` divider.
- Moved `CssBaseline` to `Theme` component.
- Improved styles in `Metrics` component.

### CODE REVIEW

File: aim/web/ui_v2/src/pages/Metrics/style.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { makeStyles } from '@material-ui/core';
+
+export default makeStyles(({ spacing }) => ({
```
Comment (roubkar) [inline aim/web/ui_v2/src/pages/Metrics/style.ts:3]: @arsengit please use more descriptive names for files.
e.g. `metricsStyles.ts`

File: aim/web/ui_v2/src/pages/Metrics/MetricsContainer.tsx [hunk 2]
```diff
@@ -1,9 +1,50 @@
...
+  }, []);
+
+  const handleResize = React.useCallback(() => {
+    document.addEventListener('mousemove', startResize);
+    document.addEventListener('mouseup', endResize);
+  }, []);
+
+  const startResize = React.useCallback((event: MouseEvent): void => {
+    const containerHeight: number =
+      (tableRef.current as HTMLDivElement)?.getBoundingClientRect()?.height +
```
Comment (roubkar) [inline aim/web/ui_v2/src/pages/Metrics/MetricsContainer.tsx:23]: @arsengit is using the `as` keyword necessary here?
Cause RefObject type is already mentioned above via generic.

**Review by roubkar — APPROVED** [review]
lgtm

**Review by gorarakelyan — APPROVED** [review]
great

**Pull request merged**